### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by AmProsius** [AmProsius/gothic-1-community-patch#336] [General]
## Roadmap
The Gothic 1 Community Patch is not dead! Many of you are wondering if and when this project will be developed further. I can't give a definitive answer to this question, because both @szapp and I have very limited time at the moment. But we constantly try to improve workflows and fix issues from time to time.

Currently we are working on automating fix templates so we can speed up the process of fixing issues.

If you are a developer yourself, consider applying to this project. If not, you are welcome to open issues or start a discussion here!

**Comment by I-Fucking-Hate-Github** [comment]
Hi, I submitted a bunch of issues a week ago only to be automatically banned for "spamming". Then a week later support told me my ban would not be overturned because I'm "not using GitHub for its intended purpose". So I'm done with this stupid website.
But before it all goes to waste, here is a copy of all the issues I opened: https://pastebin.com/XdWqtnTH

**Reply by Quintus24** [reply]
Hi, wow, I was actually wondering what happened to you, I did noticed you added some issues and then disappeared the next day. :neutral_face:
Not to mention that even your user profile was missing from GitHub, sorry that happened to you. :disappointed:

**Reply by AmProsius** [reply]
Thank you so much for saving the issues! @szapp and I were also wondering what happened, because we both got notifications and got excited, but the links were already dead.

A few of these were already taken care of, but when I find the time, I will open issues for the rest. I hope that the account misunderstanding will be sorted out though!
